### PR TITLE
Compability with Rails's main branch (7.0.0.alpha)

### PR DESCRIPTION
**Pull request by dorianmariefr**
I ran the tests and everything is green.

I updated the rails version used in the local Gemfile to the latest commit on rails/rails main branch.

I figured if you want to test against different versions of rails you could use appraisal https://github.com/thoughtbot/appraisal

Really not sure about my approach, let me know what you think ❤️

### CODE REVIEW

File: arbre.gemspec [hunk 1]
```diff
@@ -21,6 +21,6 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = '>= 2.5'
 
-  s.add_dependency("activesupport", ">= 3.0.0", "< 6.2")
+  s.add_dependency("activesupport", ">= 3.0.0", "< 7.0")
```
Comment (dorianmariefr) [inline arbre.gemspec:24]: this is the main thing

**Comment by dorianmariefr** [comment]
the tests are failing because:

> activesupport-7.0.0.alpha requires ruby version >= 2.7.0, which is incompatible

so I "downgraded" rails locally to 6.1.3

**Comment by dorianmariefr** [comment]
ok, removed the upgrades because it's failing on jruby, will do a separate PR

File: arbre.gemspec (hunk 1 above)
Comment (sunny) [inline arbre.gemspec:24]: Now that Rails 7 is released, this should probably be changed to:

```suggestion
  s.add_dependency("activesupport", ">= 3.0.0", "< 8.0")
```

**Comment by javierjulio** [comment]
We'll tackle this in #314 thanks!

**Pull request closed without merging**